bzzeth/bzzeth_test.go

```diff
@@ -17,28 +17,44 @@
 package bzzeth
 
 import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
 	"errors"
+	"io/ioutil"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethersphere/swarm/chunk"
+	"github.com/ethersphere/swarm/network"
 	p2ptest "github.com/ethersphere/swarm/p2p/testing"
+	"github.com/ethersphere/swarm/storage"
+	"github.com/ethersphere/swarm/storage/localstore"
 	"github.com/ethersphere/swarm/testutil"
 )
 
 func init() {
 	testutil.Init()
 }
 
-func newBzzEthTester() (*p2ptest.ProtocolTester, *BzzEth, func(), error) {
-	b := New()
+func newBzzEthTester(t *testing.T, prvkey *ecdsa.PrivateKey, netStore *storage.NetStore) (*p2ptest.ProtocolTester, *BzzEth, func(), error) {
```

Comment:
netstore can be changed to `chunk.Store` as mentioned

Reply:
you mean change *storage.Netstore to chunk.Store?